tests/testOpenFiles.py

```diff
@@ -68,9 +71,11 @@ class OpenFilesTest(unittest.TestCase):
 
     def setUp(self):
         self.originalLimits = resource.getrlimit(resource.RLIMIT_NOFILE)
-        print 'NOFILE rlimit:', self.originalLimits
-        resource.setrlimit(resource.RLIMIT_NOFILE, (10, self.originalLimits[1]))
-        print 'NOFILE rlimit:', resource.getrlimit(resource.RLIMIT_NOFILE)
+        # don't want this to depend on how many files e.g. py.test has open at the start.
+        currentOpenFiles = len(getOpenFiles())
+        print('NOFILE rlimit:', self.originalLimits)
+        resource.setrlimit(resource.RLIMIT_NOFILE, (currentOpenFiles+5, self.originalLimits[1]))
```

Comment:
that's because we disable "spaces around operator" warnings because it's an area where there is vagueness in PEP8 and where we disagree with some interpretations.